### PR TITLE
language update (follow-up on #1398)

### PR DESCRIPTION
**Pull request by discip**
After the behavior of the calibration was changed with #1398, I also had to adjust the corresponding entries in the languages.
But due to lack of space on the **`TS80P`** I had to postpone this until it was taken care of by @Ralim.

* Additionally, the entries have been rearranged to match the structure present in the current menu on the corresponding device.

* Also, the unused entries have been removed. (To save the trouble of translating them.)

I've tried my best to find out who contributed to each language and I hope I haven't overlooked anyone.
Please take care of the appropriate translation.

After being asked how to submit the appropriate changes, I figured the best way would be to just use the suggestion feature that already exists:
e.g:

thanks in advance

BE
@lidacity
@dmitrygribenchuk

BG
@dimitroffs

CS
@cewbdex

DA
@t-xdk

EL
@Devnol

ES
@Meldinov
@IkerGarcia

FI
@Dids

FR
@HanaO00

HR
@jonnieZG

HU
@balika011
@hippi-viking

IT
@federicodilo

LT
@Vinigas
@jvitkauskas

NL
@FaBjE
@KoopsF

NO
@LarsSimonsen

PL
@KITT3000
@playday3008
@d-rez

PT
@lfhohmann
@speleos

RO
@cipyyy
@sandmanRO

RU
@onxblog
@JugglerLKR
@playday3008

SK
@OndroNR
@gamelaster

SL
@pikipirs
@gigibu5

SR
@vomindoraan
@abrakatebra

SV
@piksel

TR
@JustNoLimit
@mesutkilic
@wortex1979
@muhammedfurkan42

UK
@BatsIhor
@playday3008
@eslng
@hrebeniukB

VI
@prokrypt

CJK
@alvinhochun
@i4M1k0SU
@skishida
@contactalexliu

### CODE REVIEW

**Comment by hippi-viking** [comment]
Thank you, but please don't spam all the referenced people with all the separate commits.

**Comment by discip** [comment]
@hippi-viking
Sorry!
I did not mean to spam anyone.
I was not aware of the fact, that every commit would cause a notification. 😓

**Comment by hippi-viking** [comment]
No worries :) I only wrote to let you know that every commit is a separate notification for everyone referenced.

**Comment by Devnol** [comment]
yeah don't worry, I've seen people do it on projects with like 14k members, absolute hell.

**Comment by discip** [comment]
I am not in charge, but if you want to update your translation, just open a PR.

**Comment by discip** [comment]
@Devnol
I am not in charge, but if you want to update your translation, just open a PR.

**Comment by discip** [comment]
@Devnol
Sorry, I think I did not get you right the first time.

If you meant to change the lines I did update, than it is best to do it like so:
I think you know the process depicted in the attached pictures.

**Comment by discip** [comment]
@Devnol
Sorry, I think I misunderstood you the first time.

If you wanted to change the lines I updated, the best way to do it is like this:
I think you know the process shown in the attached pictures.

**Comment by prokrypt** [comment]
VI translation is all @River-b, not me (:

File: Translations/translation_IT.json [hunk 1]
```diff
@@ -230,26 +287,19 @@
...
-		"TempChangeShortStep": {
+		"CalibrateCJC": {
 			"text2": [
-				"Temp passo",
-				"breve"
+				"Calibrate CJC",
+				"at next boot"
 			],
-			"desc": "Imposta il passo dei valori di temperatura per una breve pressione dei tasti"
+			"desc": "Calibrate tip Cold Junction Compensation at the next boot (not required if Delta T is < 5°C)"
```
Comment (federicodilo) [inline Translations/translation_IT.json:295]: Compensa l'errore delle rilevazioni della termocoppia al prossimo avvio (non necessario se l'errore su T è <5 °C)

**Review by federicodilo — CHANGES_REQUESTED** [review]
Feel free to change those or I'll do myself once merged

File: Translations/translation_ES.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	],
 	"tempUnitFahrenheit": false,
 	"messages": {
-		"SettingsCalibrationWarning": "¡Asegúrate que la punta esté a temperatura ambiente antes de empezar!",
+		"SettingsCalibrationWarning": "Please make sure tip & handle are at room temperature at next boot!",
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:10]:
```suggestion
		"SettingsCalibrationWarning": "¡Asegúrate que la punta esté a temperatura ambiente al inicio",

```

File: Translations/translation_ES.json [hunk 3]
```diff
@@ -66,35 +66,35 @@
 				"Potencia",
 				"ajustes"
 			],
-			"desc": "Ajustes de potencia"
+			"desc": ""
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:69]:
```suggestion
			"desc": "Ajustes de potencia"

```

File: Translations/translation_ES.json [hunk 4]
```diff
@@ -66,35 +66,35 @@
...
-			"desc": "Ajustes de potencia"
+			"desc": ""
 		},
 		"SolderingMenu": {
 			"text2": [
 				"Soldadura",
 				"ajustes"
 			],
-			"desc": "Ajustes para soldar."
+			"desc": ""
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:76]:
```suggestion
			"desc": "Ajustes para soldar"

```

File: Translations/translation_ES.json [hunk 5]
```diff
@@ -66,35 +66,35 @@
...
 		},
 		"PowerSavingMenu": {
 			"text2": [
 				"Modos de",
 				"reposo"
 			],
-			"desc": "Ajustes de ahorro energético."
+			"desc": ""
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:83]:
```suggestion
			"desc": "Ajustes de ahorro energético"

```

File: Translations/translation_ES.json [hunk 6]
```diff
@@ -66,35 +66,35 @@
...
 		},
 		"UIMenu": {
 			"text2": [
 				"Interfaz",
 				"de usuario"
 			],
-			"desc": "Ajustes de interfaz de usuario."
+			"desc": ""
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:90]:
```suggestion
			"desc": "Ajustes de interfaz de usuario"

```

File: Translations/translation_ES.json [hunk 7]
```diff
@@ -66,35 +66,35 @@
...
 		},
 		"AdvancedMenu": {
 			"text2": [
 				"Ajustes",
 				"avanzados"
 			],
-			"desc": "Ajustes avanzados."
+			"desc": ""
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:97]:
```suggestion
			"desc": "Ajustes avanzados"

```

File: Translations/translation_ES.json [hunk 8]
```diff
@@ -140,159 +203,110 @@
...
 			],
-			"desc": "Al soldar, una pulsación larga en ambos botones los bloquea (D=desactivar | B=sólo potenciar | F=bloqueo total)."
+			"desc": "Elige el límite de potencia máxima del soldador (en vatios)"
 		},
-		"MinVolCell": {
+		"CalibrateCJC": {
 			"text2": [
-				"Mínimo",
-				"voltaje"
+				"Calibrate CJC",
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:292]:
```suggestion
				"Calibrar CJC",

```

File: Translations/translation_ES.json [hunk 9]
```diff
@@ -140,159 +203,110 @@
...
+				"at next boot"
 			],
-			"desc": "voltaje mínimo permitido por célula (3S: 3 - 3,7V | 4-6S: 2,4 - 3,7V)"
+			"desc": "Calibrate tip Cold Junction Compensation at the next boot (not required if Delta T is < 5°C)"
```
Comment (IkerGarcia) [inline Translations/translation_ES.json:295]:
```suggestion
			"desc": "Calibrar compensación de punta fría en el próximo inicio (no necesario si Delta T es < 5°C)"

```

File: Translations/translation_RO.json [hunk 10]
```diff
@@ -66,35 +66,35 @@
 				"Setări de",
 				"alimentare"
 			],
-			"desc": "Setări de alimentare"
+			"desc": ""
```
Comment (cipyyy) [inline Translations/translation_RO.json:69]: "desc": "Setări de alimentare"

File: Translations/translation_RO.json [hunk 11]
```diff
@@ -66,35 +66,35 @@
...
-			"desc": "Setări de alimentare"
+			"desc": ""
 		},
 		"SolderingMenu": {
 			"text2": [
 				"Setări de",
 				"lipire"
 			],
-			"desc": "Setări de lipire"
+			"desc": ""
```
Comment (cipyyy) [inline Translations/translation_RO.json:76]: "desc": "Setări de lipire"

File: Translations/translation_RO.json [hunk 12]
```diff
@@ -66,35 +66,35 @@
...
 		},
 		"PowerSavingMenu": {
 			"text2": [
 				"Modul",
 				"repaus"
 			],
-			"desc": "Setări de economisire a energiei"
+			"desc": ""
```
Comment (cipyyy) [inline Translations/translation_RO.json:83]: "desc": "Setări de economisire a energiei"

File: Translations/translation_RO.json [hunk 13]
```diff
@@ -66,35 +66,35 @@
...
 		},
 		"UIMenu": {
 			"text2": [
 				"Interfată",
 				"utilizator"
 			],
-			"desc": "Setări interfată utilizator"
+			"desc": ""
```
Comment (cipyyy) [inline Translations/translation_RO.json:90]: "desc": "Setări interfată utilizator"

File: Translations/translation_RO.json [hunk 14]
```diff
@@ -66,35 +66,35 @@
...
 		},
 		"AdvancedMenu": {
 			"text2": [
 				"Optiuni",
 				"avansate"
 			],
-			"desc": "Optiuni avansate"
+			"desc": ""
```
Comment (cipyyy) [inline Translations/translation_RO.json:97]: "desc": "Optiuni avansate"

**Comment by discip** [comment]
@Ralim
Good morning Ben.

This got somewhat out of hand, since people do not update the files changed in this PR but rather the present ones.

I think it would be best if you merged this in before people commit their language updates.
Otherwise this misses the point of asking people to adapt to the current situation.
 thanks

**Comment by discip** [comment]
@Ralim
Good morning Ben.

This has gotten a bit out of hand as people are not updating the files changed in this PR, but the existing ones which are missing the latest information.

I think it would be best if you merged this in before people commit their language updates.

Otherwise this misses the point of asking people to adapt to the current situation.
 thanks

**Comment by discip** [comment]
That was quick! 😮

**Pull request merged**